cmd/mistryd/server.go

```diff
@@ -350,19 +352,29 @@ func (s *Server) ListenAndServe() error {
 
 // RebuildResult contains result data on the rebuild operation
 type RebuildResult struct {
-	successful  int
-	failed      int
-	pruneReport dockerTypes.ImagesPruneReport
+	successful       int
+	failed           []string
+	prunedImages     int
+	prunedContainers int
+	reclaimedSpace   uint64
 }
 
 func (r RebuildResult) String() string {
+	var failedNames string
+	if len(r.failed) > 0 {
+		failedNames = " Failed names: " + strings.Join(r.failed, ", ")
+	} else {
+		failedNames = ""
+	}
+
 	return fmt.Sprintf(
-		"Rebuilt: %d Failed: %d Pruned: %d Reclaimed: %d bytes",
-		r.successful, r.failed, len(r.pruneReport.ImagesDeleted), r.pruneReport.SpaceReclaimed)
+		"Rebuilt: %d Pruned images: %d Pruned containers: %d Reclaimed: %s Failed: %d%s",
```

Comment:
[nit] Add a separator (eg. comma) after each statistic for improved readability.